helm/druid/templates/broker/deployment.yaml

```diff
@@ -65,10 +65,16 @@ spec:
             valueFrom:  {fieldRef: {fieldPath: metadata.name}}
           - name: POD_NAMESPACE
             valueFrom: {fieldRef: {fieldPath: metadata.namespace}}
+          {{- with .Values.env }}
+            {{- toYaml . | nindent 10 }}
+          {{- end }}
           {{- range $key, $val := .Values.broker.config }}
           - name: {{ $key }}
             value: {{ $val | quote }}
-          {{- end}}
+          {{- end }}
+          {{- with .Values.broker.env }}
```

Comment:
Since there is no other thing referencing environment variables I didn't see a need to get more complicated than `env` but if its what is needed to get this merged then sure, I'm not married to the name.

Reply:
i just prefer it this way to make it clear we aren't overwriting the entire environment variables section with the extra env variable, just appending

Reply:
Yea I can see what you're saying.  I'll make the change

Reply:
@jwitko could you make a commit for this? I agree with @georgew5656.